### PR TITLE
2017-09-25 LWIMIQ

### PR DESCRIPTION
**Pull request by lgalis**
2017-09-25 LWIMIQ

### CODE REVIEW

**Comment by lgalis** [comment]
@jntullo, @syncrou - please review if you can

File: site/_posts/2017-09-25-last-week-of-summer.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: "Last Week in ManageIQ: Last week of summer ... "
+author: [lgalis](https://github.com/lgalis)
```
Comment (cybette) [inline site/_posts/2017-09-25-last-week-of-summer.md]: I don't think links are valid in the front matter. Please remove the URL and the brackets, thanks!

Comment (lgalis) [reply site/_posts/2017-09-25-last-week-of-summer.md]: @cybette  - Done

**Comment by JPrause** [comment]
Excellent! Thanks @lgalis LGTM

**Comment by jntullo** [comment]
Nice post @lgalis! LGTM 👍

**Comment by JPrause** [comment]
@cybette looks like we're good to go,..no further reviews or edits. Can you merge when able. 👍

File: site/_posts/2017-09-25-last-week-of-summer.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+The dialog generation for orchestration template was modified in [PR 15982](https://github.com/ManageIQ/manageiq/pull/15982) to rely on the templates to provide the deployment options.<br> 
+
+[PR 16011](https://github.com/ManageIQ/manageiq/pull/16011) adds 'description' to the list of automate value fields, allowing it to be updated by an automate method.<br> 
+
+## Fixed
+
+To allow the Azure SSA job to succeed, a number of timeouts and workers' memory were increased in [PR 16016](https://github.com/ManageIQ/manageiq/pull/16016).
+
+After [PR 115995](https://github.com/ManageIQ/manageiq/pull/15995), the request to destroy linked events is queued to prevent the UI hanging when deleting an instance of MiqServer.
```
Comment (cybette) [inline site/_posts/2017-09-25-last-week-of-summer.md]: minor thing: PR 15995, not 115995 (extra digit)

File: site/_posts/2017-09-25-last-week-of-summer.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## Deleted
+
+The deletion of the week, [PR16005](https://github.com/ManageIQ/manageiq/pull/16005) - removes 'format_by_(class|column)', as these are no longer used.
+
+
+## Product Tools
+
+Two very useful product tools were merged this week: 
+
+[One](https://github.com/ManageIQ/manageiq/pull/15982) to replicate a server settings to other servers.
```
Comment (cybette) [inline site/_posts/2017-09-25-last-week-of-summer.md]: Should this be [PR 15990](https://github.com/ManageIQ/manageiq/pull/15990)?

Comment (lgalis) [reply site/_posts/2017-09-25-last-week-of-summer.md]: @cybette  - fixed

File: site/_posts/2017-09-25-last-week-of-summer.md (hunk 2 above)
Comment (lgalis) [reply site/_posts/2017-09-25-last-week-of-summer.md]: @cybette - fixed

File: site/_posts/2017-09-25-last-week-of-summer.md (hunk 3 above)
Comment (lgalis) [reply site/_posts/2017-09-25-last-week-of-summer.md]: @cybette - yes, thanks for noticing, fixed

**Pull request merged**